test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetListPackageTests.cs

```diff
@@ -435,6 +436,56 @@ public async Task DotnetListPackage_ProjectWithInitialTargets_Succeeds()
             }
         }
 
+        [PlatformTheory(Platform.Windows, Skip = "Enabled after .NET CLI integration for https://github.com/NuGet/Home/issues/9600")]
+        [InlineData("", false)]
+        [InlineData("--verbosity minimal", false)]
+        [InlineData("--verbosity normal", true)]
+        public void DotnetListPackage_VerbositySwitchTogglesHttpVisibility(string args, bool showsHttp)
```

Comment:
did you verify this test will actually pass? :D Is the plan to fix it later?

Reply:
I am doing that now.

Reply:
I had to fix some things up and put in some temporary hacks to point to a private CLI and private NuGet build, but it worked out.

Reply:
YEp, have had to do the same thing. 

@martinrrm is gonna try to do the same with the `--prerelease` option. 

He might shoot you some questions.